src/engine/Resources/Sound.ts

```diff
@@ -274,8 +276,15 @@ export class Sound implements ILoadable, IAudio {
 
    /**
     * Play the sound, returns a promise that resolves when the sound is done playing
+    * Valid string inputs include:
+    * -'linearUp'
+    * -'linearDown'
+    * -'ease'
+    * -'ease-in'
+    * -'ease-out'
+    * -'ease-in-out'
     */
-   public play(volume?: string): Promise<boolean> {
+   public play(data?: string | number | IBezierPoints): Promise<boolean> {
```

Comment:
Well, they can't be "types" per se, they'd have to be constants. And then you'd have to do something like `ex.Audio.EASE_LINEAR_UP` or something. I feel like regular strings is probably OK, folks with TS will get compile errors and JS users will reference the docs to know what are valid strings to pass.